server/compaction-coordinator/src/main/java/org/apache/accumulo/coordinator/CompactionCoordinator.java

```diff
@@ -653,6 +679,21 @@ protected void startFailureSummaryLogging() {
     ThreadPools.watchNonCriticalScheduledTask(future);
   }
 
+  private <T> void printFailures(String logPrefix,
+      ConcurrentHashMap<T,FailureCounts> failureCounts) {
+    for (var key : failureCounts.keySet()) {
+      failureCounts.compute(key, (k, counts) -> {
+        if (counts != null && counts.failures > 0) {
```

Comment:
Do you think it would be useful to print the summary if only successes and no failures? I'm wondering if we should always print the summary and change the method name from `printFailures` to `printSummary`. Do you think that's useful in practice?

Reply:
Made updates in 5ee4434 for this.   It would be interesting to see the per table and per queue success counts.  The per compactor success counts could be numerous and not that interesting, so put those at trace.

Reply:
Also tried using the function name printSummary but noticed the class had a good bit of stuff w/ the naming summary that is unrelated to this.  So named it printStats